app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
Oh, ok, sorry, I wasn't clear on what I actually mean 🙈 It does not *necessarily*  need to be especially this certain exception. I just meant: if someone not being a student or on a team can access the controller action, the resulting error should somehow make clear what's wrong - and a 500 status could be anything 😉 

The way you put it now, sure it works, but it looks kinda weird: having the `student_team` fetched in two places and then explicitly raising the exception appears wrong in a controller, don't you think?

In which context is this controller used? If it's only used by the user actually owning the `attendance`, it could be simplified to
```ruby
def attendance
  current_user.attendances.find(params[:id])
end
```

If the user is also changing his or her teammates attendances, then one could maybe work with the abilities? Would look somewhat like this I think:
```ruby
class AttendancesController < ApplicationController
  load_and_authorize_resource

  def update
    @attendance.update_attributes!(attendance_params)
    redirect_back fallback_location: user_path(current_user), notice: 'Ok, successfully updated'
  end

  def destroy
    @attendance.destroy
    redirect_back fallback_location: user_path(current_user), notice: 'Ok, the ticket was added back to the pool of available tickets'
  end

  private

  def attendance_params
    params.require(:attendance).permit(:confirmed)
  end
end
```

I would of course prefer the first scenario :v: